app/main.py

```diff
@@ -1,13 +1,22 @@
-def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
-
-
-def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
-
-
-def format_linter_report(linter_report: dict) -> list:
-    # write your code here
-    pass
+def format_linter_error(error: dict) -> dict:
+    return {
+        "line": error["line_number"], "column": error["column_number"],
+        "message": error["text"], "name": error["code"], "source": "flake8"

```

Comment:
Bad formatting, you should try to follow 1 of the structures:
1 - all in 1 line (not good here, will exceed the line len)
2 - 1 by 1 in a new line